contracts/Tournament.sol

```diff
@@ -2,25 +2,44 @@
 pragma solidity ^0.8.10;
 
 contract Tournament {
+
+    enum State { Ongoing, BetsClosed, Failed, PaidOut }
+
     uint public immutable COMMISSION = 5;
     
     address public dealer;
     uint256 public buyIn;
+    uint256 public id;
```

Comment:
What do we need that field for?

Reply:
https://github.com/budziam/cobet/pull/1?fbclid=IwAR0SKThXBg8Kd6JALVTg2BKrxmBJ2cndzKX-fIhcF1WzPYB_Aewu0mTs9Uc#discussion_r804690382

Reply:
deleted